app/models/game.rb

```diff
@@ -0,0 +1,4 @@
+class Game < ApplicationRecord
+  belongs_to :player
```

Comment:
A game will need two players, one white and one black. Maybe add something like this instead:

```
belongs_to :black_player
belongs_to :white_player
```